kubectl-rook-ceph.sh

```diff
@@ -446,6 +446,11 @@ function run_start_debug() {
   # scale the deployment to 0
   KUBECTL_NS_CLUSTER scale deployments "$deployment_name" --replicas=0
 
+  # wait for the deployment pod to be deleted
+  deployment_pod=$(KUBECTL_NS_CLUSTER get pod | grep "$deployment_name" | awk '{ print $1  }')
```

Comment:
Does this line need to be moved before line 447? Otherwise we might not detect the pod name if it stops too quickly before this command.